pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -849,60 +821,60 @@ def mc_cal_wf_errs(num_mc_injs, inj_dists, cal_err, wf_err, max_dc_cal_err):
     return inj_dist_mc
 
 
-def read_multiinspiral_timeslides_from_files(file_list):
-    """
-    Read time-slid multiInspiral tables from a list of files
-    """
-
-    multis = None
-    time_slides = []
-
-    contenthandler = glsctables.use_in(LIGOLWContentHandler)
-    for this_file in file_list:
-        doc = utils.load_filename(this_file, compress='auto',
-                                  contenthandler=contenthandler)
-
-        # Extract the time slide table
-        time_slide_table = \
-            Table.get_table(doc, lsctables.TimeSlideTable.tableName)
-        slide_mapping = {}
-        curr_slides = {}
-        for slide in time_slide_table:
-            curr_id = int(slide.time_slide_id)
-            if curr_id not in curr_slides:
-                curr_slides[curr_id] = {}
-                curr_slides[curr_id][slide.instrument] = slide.offset
-            elif slide.instrument not in curr_slides[curr_id]:
-                curr_slides[curr_id][slide.instrument] = slide.offset
-
-        for slide_id, offset_dict in curr_slides.items():
-            try:
-                # Is the slide already in the list and where?
-                offset_index = time_slides.index(offset_dict)
-                slide_mapping[slide_id] = offset_index
-            except ValueError:
-                # If not then add it
-                time_slides.append(offset_dict)
-                slide_mapping[slide_id] = len(time_slides) - 1
-
-        # Extract the multi inspiral table
-        try:
-            multi_inspiral_table = Table.get_table(doc, 'multi_inspiral')
-            # Remap the time slide IDs
-            for multi in multi_inspiral_table:
-                new_id = slide_mapping[int(multi.time_slide_id)]
-                multi.time_slide_id = gilwdchar(
-                                      f"time_slide:time_slide_id:{new_id}")
-            if multis:
-                multis.extend(multi_inspiral_table)
-            else:
-                multis = multi_inspiral_table
-        except Exception as exc:
-            err_msg = "Unable to read a time-slid multiInspiral table "
-            err_msg += f"from {this_file}."
-            raise RuntimeError(err_msg) from exc
-
-    return multis, time_slides
+# def read_multiinspiral_timeslides_from_files(file_list):
```

Comment:
Remove all of this if no loner needed.

Reply:
This will have to be handled with hdf so it can definitely go.